### DISCUSSION TITLE
Update Php7 ->8x

### DISCUSSION THREAD

**Discussion by antonov124** [wavelog/wavelog#434] [Q&A] [answered]
Ich habe auf meiner VM mit aktuell Debian 11.9 von Php 7.4.33 versucht das Update auf Php 8.x zu machen  quelle: https://php.watch/articles/php-8.3-install-upgrade-on-debian-ubuntu     das klappte auch soweit. nur habe ich dann nur noch ein wirrwar an anzeigen auf der startseite von wavelog gehabt.  Ich hatte mir vorher einen Snapshot angelegt für den Fall der Fälle welchen ich dann zurückgespielt habe.  Besteht den die Möglichkeit vom Update innerhalb Wavelog? Oder schlägt das so oder so fehl?
gruß Rainer

**Comment by HB9HIL** [comment, marked as answer]
Hallo Rainer

Innerhalb Wavelog kannst du PHP nicht updaten. Wavelog bedient sich nur des installierten PHP's über den Webserver. 
Du kannst kontrollieren welche PHP Version dein Webserver aktuell verwendet, in dem du unter Admin > Debug schaust welche PHP Version hier steht. 

<img width="488" alt="image" src="https://github.com/wavelog/wavelog/assets/80885850/690d0fb8-6762-4bba-8afb-3dc32895912f">

**Reply by antonov124** [reply]
Ja innerhalb ist mir bewußt, dann hab ich mich falsch ausgedrückt.
  | 7.4.33 Deprecated    steht im Debug

Mir ging es nur darum ob ich nach dem Update auf 8.x etwas in Wavelog anpassen/editieren muss, oder Wavelog normalerweise den Sprung "verkraften" sollt

**Reply by HB9HIL** [reply]
Wavelog "verkraftet" das. Es scheint als sei bei dir eher dann was schief gelaufen.

**Reply by antonov124** [reply]
Okay dann such ich mir mal ein PHP Spezi ;) also falls du jemand kennen solltest?

**Reply by HB9HIL** [reply]
Kannst mir mal ne E-Mail schreiben. E-Mail siehe qrz.com
